src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Logo from '../assets/turtle_header.jpeg'
+import '../App.css'
+class Header extends Component {
+    render() {
+        return (
+            <div className ="header">
+                <img className = "logo" src={Logo} alt="picture of a turtle" />
+                <h3> Our Header! </h3>
```

Comment:
for the most part your spacing looks really good. I would just clean up this file. Typically you don't have spaces around the equal sign of attributes.

Reply:
Fixed!